### PR TITLE
including only the baseline models notebooks

### PR DESCRIPTION
**Pull request by tikeswar**
I have cleaned my branch, keeping only the baseline models notebooks. Please take a look and merge. @MichalRoztocki @zunicd

### CODE REVIEW

**Comment by MichalRoztocki** [comment]
I just pushed a commit to this branch with some changes because I couldn't comment on the notebook via github.

BOW notebooks:

1. Changed title in first markdown cell of each BOW notebook so that the name of each type of protein was correct.
2. Changed path to the data folder to match what Damir and I have been using since the beginning, to maintain a single folder structure.
3. Removed "In Tensorflow" from title because it was unnecessary.
4. Removed a few redundant df.head's to clean up.
5. Didn't change this, but I think you could just add activation='relu' inside the Dense layer instead of importing it as a layer and adding it separately. If you decide to change it to that, make sure to remove the import of the Activation layer in the first code cell as well.
6. Got rid of y_train_pred = model.predict(X_train).round().astype(int) in your cross-validation loop because you never used it after assigning it and it's not useful anyway.
7. I didn't remove it, but I think you should reconsider taking the model with the highest validation accuracy out of the list of models you made for ACP and DBP. It's not really what cross-validation is for - each model is trained on a subset of the data, and validated on another subset; a higher validation accuracy on one model doesn't mean a better model, since each of the models is the same, with the same hyperparameters. If you want to evaluate your model's performance on the test data, you should retrain a model with the hyperparameters you decided on, after cross-validation, on the entire train set and then check the performance of that model on the test set.

SGT notebooks: 

1. Changed the title of each notebook in the same way as with the BOW notebooks.
2. Changed file paths as in BOW notebooks.
3. Removed the df.sample()'s to clean up, not that helpful after doing df.head.
4. Removed the "in Tensorflow" just like in the BOW notebooks
5. Didn't remove this, but not sure why you added an ID for DBP before doing deep learning when it already has a code.
6. Same as with BOW, removed y_train_pred = model.predict(X_train).round().astype(int) from AMP SGT notebook because it wasn't used.
7. Same concern as above with taking the model with the highest validation accuracy out of the cross-validation loop rather than retraining with the same hyperparameters on the full train set and then evaluating performance on test set.

I'm also not sure if label encoding the labels makes sense, since they're already in binary form. You did this in both the BOW and SGT notebooks, so if label encoding leads to the same result, I think you should remove that as well.

**Comment by tikeswar** [comment]
@MichalRoztocki 
Looks good with the changes you have made.

1. In the modeling: I first tried to exactly replicate the original author's paper, keeping much of his code intact. Once I was able to replicate his result, I just tried to adapt to our data-set. Adding the 'relu' in a separate layer, label encoding, were all just carried over from the original SGT. But yes, I could have just specified the 'relu' in the Dense layer itself, and simply used np.array(y) instead of the label encoder.
2. Also I get your point on the k-fold cross-validation. I could just report the average accuracy from the k-fold, but I am thinking it should be ok the way I have done it, and this is why. Just like 'early stopping' does not give a better _model_, but it gives better _weights_ for the given model. Similar to that ... choosing the 'best' of the k-fold I am just taking the better weights of the same model (so in some sense this would be the most optimistic case).

But that said I can take care of these ...
1. Move the 'relu' to the dense layer itself
2. simply use np.array instead of the label encoder 
3. end the notebook once I get the average accuracy from the k-fold
4. Yes, there was no need of a separate ID for DBP (did that in a rush thinking the code had dupliccates)

**Comment by MichalRoztocki** [comment]
Ok, I understand then. 
1. That's fine, it's not really a big deal so you can leave it if you want.
2. I would change this I think, label encoding just doesn't make sense in our situation, and it's a roundabout way to get an np array.
3. I wouldn't say you have to end the notebook there, but again, retraining on the full train set with the same model would make sense. It's not exactly like early stopping, since that's a method to avoid overfitting, whereas cross validation is a method to pick the right hyperparameters for your model and validate across different folds. The resulting accuracy might be the same after retraining on the full dataset, but it just doesn't really make sense to evaluate a model that was trained on a random subset (fold) of the train data. I think generally speaking we don't even really use cross-validation for deep learning, but in this case I think the best thing to do at this stage is to just retrain on the full data right after getting the cross validation metrics.
4. No worries!

I'll be good with merging after these changes.

**Comment by tikeswar** [comment]
I had to create another branch as I was not able to resolve some git pull/push issue on this branch ... pls take a look at the baseline2 branch
